recursive_art.py

```diff
@@ -43,8 +56,24 @@ def evaluate_random_function(f, x, y):
         >>> evaluate_random_function(["y"],0.1,0.02)
         0.02
     """
-    # TODO: implement this
-    pass
+    if len(f)==1:
```

Comment:
I recommend that you add one or two unit tests of your own to test the function.